## 78% of end-end tests have passed
```
Total executed tests: 51
Failed tests: 11
Expected to fail tests: 0
Passed tests: 40
```
```
IDs of failed tests: 727231,702855,702730,702745,702731,702733,702813,702859,702782,703391,703503 
```
<h3>Failed tests (11)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=feat/cache-offline-data&PR_ID=21070&APK_NAME=21070.apk&TR_CASE_IDS=727231,702855,702730,702745,702731,702733,702813,702859,702782,703391,703503">Rerun failed tests</a></li><br/><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641981">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: Button </div><div>Device 1: Find `Button` by `accessibility id`: `account-emoji` </div></blockquote></p>

```
critical/test_wallet.py:240: in test_wallet_add_remove_regular_account
    self.wallet_view.remove_account(account_name=new_account_name)
../views/wallet_view.py:176: in remove_account
    self.account_emoji_button.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:84: in find_element
    raise exception
../views/base_element.py:77: in find_element
    return self.driver.find_element(self.by, self.locator)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:409: in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E       at UIA2Proxy.command (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at processTicksAndRejections (node:internal/process/task_queues:95:5)
E       at AndroidUiautomator2Driver.helpers.doFindElementOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/lib/commands/find.js:30:10)
E       at doFind (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:36:17)
E       at wrappedCondFn (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:137:14)
E       at spin (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:221:20)
E       at waitForCondition (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:238:10)
E       at AndroidUiautomator2Driver.implicitWaitForCondition (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:139:12)
E       at AndroidUiautomator2Driver.findElOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:60:5)
E       at AndroidUiautomator2Driver.findElOrElsWithProcessing (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:60:12)
E       at AndroidUiautomator2Driver.findElement (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:75:12)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c8ed59419b9f45dc9c2e5b7c9610c6eb?auth=650b51cd58262a8a92ba94cc9c8f3814#39">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641975">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `communities-home-discover-card` </div><div>Device 1: Tap on found: Button </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:262: in test_community_discovery
    self.home.community_card_item.wait_for_elements(seconds=120)
../views/base_element.py:138: in wait_for_elements
    raise TimeoutException(
 Device 1:  BaseElement by accessibility id:`community-card-item` is not found on the screen after wait_for_elements
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c265581dc94d4f829b5373903fc0032f?auth=4b479aa06d02c34d44cf059821169b42#355">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641958">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_1_1_public_chats.py:20: in prepare_devices
    self.drivers, self.loop = create_shared_drivers(2)
base_test_case.py:309: in create_shared_drivers
    drivers = loop.run_until_complete(start_threads(test_suite_data.current_test.name,
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:26: in start_threads
    returns[k] = await returns[k]
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:257: in __init__
    super().__init__(
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:206: in __init__
    self.start_session(capabilities)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:346: in start_session
    response = self.execute(RemoteCommand.NEW_SESSION, w3c_caps)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py:229: in check_response
    raise exception_class(message, screen, stacktrace)
 Sauce could not start your job. For more information on what happened, please visit https://app.eu-central-1.saucelabs.com/tests/e1993c5be98f4b73a4578ac09cb3d5d9
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641932">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_1_1_public_chats.py:20: in prepare_devices
    self.drivers, self.loop = create_shared_drivers(2)
base_test_case.py:309: in create_shared_drivers
    drivers = loop.run_until_complete(start_threads(test_suite_data.current_test.name,
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:26: in start_threads
    returns[k] = await returns[k]
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:257: in __init__
    super().__init__(
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:206: in __init__
    self.start_session(capabilities)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:346: in start_session
    response = self.execute(RemoteCommand.NEW_SESSION, w3c_caps)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py:229: in check_response
    raise exception_class(message, screen, stacktrace)
 Sauce could not start your job. For more information on what happened, please visit https://app.eu-central-1.saucelabs.com/tests/e1993c5be98f4b73a4578ac09cb3d5d9
```

<br/><br/></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641937">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_1_1_public_chats.py:20: in prepare_devices
    self.drivers, self.loop = create_shared_drivers(2)
base_test_case.py:309: in create_shared_drivers
    drivers = loop.run_until_complete(start_threads(test_suite_data.current_test.name,
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:26: in start_threads
    returns[k] = await returns[k]
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:257: in __init__
    super().__init__(
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:206: in __init__
    self.start_session(capabilities)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:346: in start_session
    response = self.execute(RemoteCommand.NEW_SESSION, w3c_caps)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py:229: in check_response
    raise exception_class(message, screen, stacktrace)
 Sauce could not start your job. For more information on what happened, please visit https://app.eu-central-1.saucelabs.com/tests/e1993c5be98f4b73a4578ac09cb3d5d9
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641933">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_1_1_public_chats.py:20: in prepare_devices
    self.drivers, self.loop = create_shared_drivers(2)
base_test_case.py:309: in create_shared_drivers
    drivers = loop.run_until_complete(start_threads(test_suite_data.current_test.name,
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:26: in start_threads
    returns[k] = await returns[k]
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:257: in __init__
    super().__init__(
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:206: in __init__
    self.start_session(capabilities)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:346: in start_session
    response = self.execute(RemoteCommand.NEW_SESSION, w3c_caps)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py:229: in check_response
    raise exception_class(message, screen, stacktrace)
 Sauce could not start your job. For more information on what happened, please visit https://app.eu-central-1.saucelabs.com/tests/e1993c5be98f4b73a4578ac09cb3d5d9
```

<br/><br/></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641935">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_1_1_public_chats.py:20: in prepare_devices
    self.drivers, self.loop = create_shared_drivers(2)
base_test_case.py:309: in create_shared_drivers
    drivers = loop.run_until_complete(start_threads(test_suite_data.current_test.name,
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:26: in start_threads
    returns[k] = await returns[k]
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:257: in __init__
    super().__init__(
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:206: in __init__
    self.start_session(capabilities)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:346: in start_session
    response = self.execute(RemoteCommand.NEW_SESSION, w3c_caps)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py:229: in check_response
    raise exception_class(message, screen, stacktrace)
 Sauce could not start your job. For more information on what happened, please visit https://app.eu-central-1.saucelabs.com/tests/e1993c5be98f4b73a4578ac09cb3d5d9
```

<br/><br/></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641947">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_1_1_public_chats.py:20: in prepare_devices
    self.drivers, self.loop = create_shared_drivers(2)
base_test_case.py:309: in create_shared_drivers
    drivers = loop.run_until_complete(start_threads(test_suite_data.current_test.name,
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:26: in start_threads
    returns[k] = await returns[k]
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:257: in __init__
    super().__init__(
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:206: in __init__
    self.start_session(capabilities)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:346: in start_session
    response = self.execute(RemoteCommand.NEW_SESSION, w3c_caps)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py:229: in check_response
    raise exception_class(message, screen, stacktrace)
 Sauce could not start your job. For more information on what happened, please visit https://app.eu-central-1.saucelabs.com/tests/e1993c5be98f4b73a4578ac09cb3d5d9
```

<br/><br/></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641940">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_1_1_public_chats.py:20: in prepare_devices
    self.drivers, self.loop = create_shared_drivers(2)
base_test_case.py:309: in create_shared_drivers
    drivers = loop.run_until_complete(start_threads(test_suite_data.current_test.name,
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:26: in start_threads
    returns[k] = await returns[k]
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:257: in __init__
    super().__init__(
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:206: in __init__
    self.start_session(capabilities)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:346: in start_session
    response = self.execute(RemoteCommand.NEW_SESSION, w3c_caps)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py:229: in check_response
    raise exception_class(message, screen, stacktrace)
 Sauce could not start your job. For more information on what happened, please visit https://app.eu-central-1.saucelabs.com/tests/e1993c5be98f4b73a4578ac09cb3d5d9
```

<br/><br/></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641972">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_1_1_public_chats.py:20: in prepare_devices
    self.drivers, self.loop = create_shared_drivers(2)
base_test_case.py:309: in create_shared_drivers
    drivers = loop.run_until_complete(start_threads(test_suite_data.current_test.name,
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:26: in start_threads
    returns[k] = await returns[k]
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:257: in __init__
    super().__init__(
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:206: in __init__
    self.start_session(capabilities)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:346: in start_session
    response = self.execute(RemoteCommand.NEW_SESSION, w3c_caps)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py:229: in check_response
    raise exception_class(message, screen, stacktrace)
 Sauce could not start your job. For more information on what happened, please visit https://app.eu-central-1.saucelabs.com/tests/e1993c5be98f4b73a4578ac09cb3d5d9
```

<br/><br/></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641959">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `EmojisNumber` by `xpath`: `//*[starts-with(@text,'reply to image')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']/../..//*[@content-desc='emoji-reaction-1']/android.widget.TextView[2]` </div><div>Device 2: Element EmojisNumber text is equal to 1 </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:529: in test_community_one_image_send_reply
    self.errors.verify_no_errors()
base_test_case.py:191: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Not expected image is shown to the receiver
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c013da757e3449d3a32d4d3dcbe21421?auth=dec7bdc08cee3780a547ea3c48b8fcb5#258">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/eba86876056c4be49f9659ab1a2ed182?auth=05fb243b6af0c60abeac5a296cfae9cc#228">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (40)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641954">test_community_leave</a>, id: 702845 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/783a07cfc66f43cc9b834acec89aa8ee?auth=c8dea0130bfb80c42d032e74422f4840#306">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/86c746a0733e4e40b8a9f89eba75050c?auth=fdfeec5f8abdc34a2e7a2dc3f49cb552#275">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641943">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/783a07cfc66f43cc9b834acec89aa8ee?auth=c8dea0130bfb80c42d032e74422f4840#141">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/86c746a0733e4e40b8a9f89eba75050c?auth=fdfeec5f8abdc34a2e7a2dc3f49cb552#115">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641946">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/783a07cfc66f43cc9b834acec89aa8ee?auth=c8dea0130bfb80c42d032e74422f4840#190">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/86c746a0733e4e40b8a9f89eba75050c?auth=fdfeec5f8abdc34a2e7a2dc3f49cb552#173">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641963">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/783a07cfc66f43cc9b834acec89aa8ee?auth=c8dea0130bfb80c42d032e74422f4840#309">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/86c746a0733e4e40b8a9f89eba75050c?auth=fdfeec5f8abdc34a2e7a2dc3f49cb552#297">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641976">test_community_join_when_node_owner_offline</a>, id: 703629 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/783a07cfc66f43cc9b834acec89aa8ee?auth=c8dea0130bfb80c42d032e74422f4840#508">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/86c746a0733e4e40b8a9f89eba75050c?auth=fdfeec5f8abdc34a2e7a2dc3f49cb552#480">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641977">test_links_open_universal_links_from_chat</a>, id: 704613 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2df181bedf5f4a3b9166a6e24fa87fd3?auth=0a4a42b996d4dd62c33a6c959b479430#61">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641938">test_links_deep_links_profile</a>, id: 702775 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2df181bedf5f4a3b9166a6e24fa87fd3?auth=0a4a42b996d4dd62c33a6c959b479430#124">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641982">test_deep_links_communities</a>, id: 739307 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2df181bedf5f4a3b9166a6e24fa87fd3?auth=0a4a42b996d4dd62c33a6c959b479430#194">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641965">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e25dd2a92926483193cac19acd8f2c00?auth=21635b8cc1496e4c903d79f8218c3e45#170">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f6412d1c7d7646e9ae6e4167f0ed2200?auth=961121de4d0d71d26538f87cd9e6c62d#117">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641964">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e25dd2a92926483193cac19acd8f2c00?auth=21635b8cc1496e4c903d79f8218c3e45#129">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f6412d1c7d7646e9ae6e4167f0ed2200?auth=961121de4d0d71d26538f87cd9e6c62d#101">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641974">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0a02351abd3449a2aa91425c577a6c32?auth=e4c4960008e6f354bc7684eb8d048a5f#111">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6da33734dce8489088a4e9ce7faf9c19?auth=951a5d52fce24427351773d604284666#118">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641941">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0a02351abd3449a2aa91425c577a6c32?auth=e4c4960008e6f354bc7684eb8d048a5f#101">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6da33734dce8489088a4e9ce7faf9c19?auth=951a5d52fce24427351773d604284666#90">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641942">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0a02351abd3449a2aa91425c577a6c32?auth=e4c4960008e6f354bc7684eb8d048a5f#177">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6da33734dce8489088a4e9ce7faf9c19?auth=951a5d52fce24427351773d604284666#131">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641936">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c265581dc94d4f829b5373903fc0032f?auth=4b479aa06d02c34d44cf059821169b42#77">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641955">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c265581dc94d4f829b5373903fc0032f?auth=4b479aa06d02c34d44cf059821169b42#53">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641967">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c265581dc94d4f829b5373903fc0032f?auth=4b479aa06d02c34d44cf059821169b42#230">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641960">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c265581dc94d4f829b5373903fc0032f?auth=4b479aa06d02c34d44cf059821169b42#106">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641971">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c265581dc94d4f829b5373903fc0032f?auth=4b479aa06d02c34d44cf059821169b42#125">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641957">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b3acbb1ea1e44a178845806be7365fa5?auth=4e244d1eaf8e57fffea8394bd83a2cd4#105">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1b92ddac27a840e88145c5f5bbd72ee7?auth=c0d3596fd5ec6f4f84617638f1320ea2#118">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641956">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b3acbb1ea1e44a178845806be7365fa5?auth=4e244d1eaf8e57fffea8394bd83a2cd4#62">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1b92ddac27a840e88145c5f5bbd72ee7?auth=c0d3596fd5ec6f4f84617638f1320ea2#56">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641939">test_add_contact_field_validation</a>, id: 702777 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b3acbb1ea1e44a178845806be7365fa5?auth=4e244d1eaf8e57fffea8394bd83a2cd4#140">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1b92ddac27a840e88145c5f5bbd72ee7?auth=c0d3596fd5ec6f4f84617638f1320ea2#206">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641980">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641979">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641969">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5a25abf76898493da14c4aae037ac069?auth=3d1adfeebf0cac7c5379fd3287fbc0b4#179">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9340b990cac7441d96679b71c8ec326d?auth=834d2478072b2c4649d0e407cfa74e74#112">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/04c38ef271324150b65f795f2c1d3fe2?auth=f63aa9a234d7bde293dc12bb57aaf29b#92">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641944">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5a25abf76898493da14c4aae037ac069?auth=3d1adfeebf0cac7c5379fd3287fbc0b4#162">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9340b990cac7441d96679b71c8ec326d?auth=834d2478072b2c4649d0e407cfa74e74#84">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/04c38ef271324150b65f795f2c1d3fe2?auth=f63aa9a234d7bde293dc12bb57aaf29b#84">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641945">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5a25abf76898493da14c4aae037ac069?auth=3d1adfeebf0cac7c5379fd3287fbc0b4#326">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9340b990cac7441d96679b71c8ec326d?auth=834d2478072b2c4649d0e407cfa74e74#209">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/04c38ef271324150b65f795f2c1d3fe2?auth=f63aa9a234d7bde293dc12bb57aaf29b#207">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641934">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5a25abf76898493da14c4aae037ac069?auth=3d1adfeebf0cac7c5379fd3287fbc0b4#359">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9340b990cac7441d96679b71c8ec326d?auth=834d2478072b2c4649d0e407cfa74e74#236">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/04c38ef271324150b65f795f2c1d3fe2?auth=f63aa9a234d7bde293dc12bb57aaf29b#234">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641970">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5a25abf76898493da14c4aae037ac069?auth=3d1adfeebf0cac7c5379fd3287fbc0b4#286">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9340b990cac7441d96679b71c8ec326d?auth=834d2478072b2c4649d0e407cfa74e74#174">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/04c38ef271324150b65f795f2c1d3fe2?auth=f63aa9a234d7bde293dc12bb57aaf29b#147">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641973">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5a25abf76898493da14c4aae037ac069?auth=3d1adfeebf0cac7c5379fd3287fbc0b4#479">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9340b990cac7441d96679b71c8ec326d?auth=834d2478072b2c4649d0e407cfa74e74#279">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/04c38ef271324150b65f795f2c1d3fe2?auth=f63aa9a234d7bde293dc12bb57aaf29b#248">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641962">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/28c1960a15404c83a1222e15c1b011c0?auth=e237ebe9e4172f9b0c96af181016efff#129">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5ed4420e15744e70a92f0904ce3749ba?auth=7f3a5b43034bbb737f8facd67bdbbbcb#99">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641950">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c013da757e3449d3a32d4d3dcbe21421?auth=dec7bdc08cee3780a547ea3c48b8fcb5#306">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/eba86876056c4be49f9659ab1a2ed182?auth=05fb243b6af0c60abeac5a296cfae9cc#284">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641961">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c013da757e3449d3a32d4d3dcbe21421?auth=dec7bdc08cee3780a547ea3c48b8fcb5#425">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/eba86876056c4be49f9659ab1a2ed182?auth=05fb243b6af0c60abeac5a296cfae9cc#394">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641966">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c013da757e3449d3a32d4d3dcbe21421?auth=dec7bdc08cee3780a547ea3c48b8fcb5#552">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/eba86876056c4be49f9659ab1a2ed182?auth=05fb243b6af0c60abeac5a296cfae9cc#470">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641953">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c013da757e3449d3a32d4d3dcbe21421?auth=dec7bdc08cee3780a547ea3c48b8fcb5#330">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/eba86876056c4be49f9659ab1a2ed182?auth=05fb243b6af0c60abeac5a296cfae9cc#303">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641951">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c013da757e3449d3a32d4d3dcbe21421?auth=dec7bdc08cee3780a547ea3c48b8fcb5#395">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/eba86876056c4be49f9659ab1a2ed182?auth=05fb243b6af0c60abeac5a296cfae9cc#364">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641949">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c013da757e3449d3a32d4d3dcbe21421?auth=dec7bdc08cee3780a547ea3c48b8fcb5#202">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/eba86876056c4be49f9659ab1a2ed182?auth=05fb243b6af0c60abeac5a296cfae9cc#136">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641948">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c013da757e3449d3a32d4d3dcbe21421?auth=dec7bdc08cee3780a547ea3c48b8fcb5#160">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/eba86876056c4be49f9659ab1a2ed182?auth=05fb243b6af0c60abeac5a296cfae9cc#107">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641978">test_community_edit_delete_message_when_offline</a>, id: 704615 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c013da757e3449d3a32d4d3dcbe21421?auth=dec7bdc08cee3780a547ea3c48b8fcb5#587">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/eba86876056c4be49f9659ab1a2ed182?auth=05fb243b6af0c60abeac5a296cfae9cc#496">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641952">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c013da757e3449d3a32d4d3dcbe21421?auth=dec7bdc08cee3780a547ea3c48b8fcb5#178">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/eba86876056c4be49f9659ab1a2ed182?auth=05fb243b6af0c60abeac5a296cfae9cc#124">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641968">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c013da757e3449d3a32d4d3dcbe21421?auth=dec7bdc08cee3780a547ea3c48b8fcb5#233">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/eba86876056c4be49f9659ab1a2ed182?auth=05fb243b6af0c60abeac5a296cfae9cc#165">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/src/status_im/contexts/wallet/common/utils.cljs
+++ b/src/status_im/contexts/wallet/common/utils.cljs
@@ -7,6 +7,7 @@
             [utils.number :as number]
             [utils.string]))
 
+(def missing-price-decimals 6) ; if we don't have the monetary value of the token, we default to 6 decimals
 (defn get-first-name
   [full-name]
   (first (string/split full-name #" ")))
@@ -66,16 +67,20 @@
 (defn get-standard-crypto-format
   "For full details: https://github.com/status-im/status-mobile/issues/18225"
   [{:keys [market-values-per-currency]} token-units]
-  (if (or (nil? token-units)
-          (nil? market-values-per-currency)
-          (money/equal-to token-units 0))
-    "0"
-    (let [price          (-> market-values-per-currency :usd :price)
-          one-cent-value (if (pos? price) (/ 0.01 price) 0)
-          decimals-count (calc-max-crypto-decimals one-cent-value)]
-      (if (< token-units one-cent-value)
-        (str "<" (number/remove-trailing-zeroes (.toFixed one-cent-value decimals-count)))
-        (number/remove-trailing-zeroes (.toFixed token-units decimals-count))))))
+  (cond (or (nil? token-units)
+            (money/equal-to token-units 0))
+        "0"
+
+        (nil? (-> market-values-per-currency :usd :price))
+        (number/remove-trailing-zeroes (.toFixed token-units missing-price-decimals))
+
+        :else
+        (let [price          (-> market-values-per-currency :usd :price)
+              one-cent-value (if (pos? price) (/ 0.01 price) 0)
+              decimals-count (calc-max-crypto-decimals one-cent-value)]
+          (if (< token-units one-cent-value)
+            (str "<" (number/remove-trailing-zeroes (.toFixed one-cent-value decimals-count)))
+            (number/remove-trailing-zeroes (.toFixed token-units decimals-count))))))
 
 (defn get-market-value
   [currency {:keys [market-values-per-currency]}]

--- a/src/status_im/contexts/wallet/common/utils_test.cljs
+++ b/src/status_im/contexts/wallet/common/utils_test.cljs
@@ -64,6 +64,11 @@
       (is (= (utils/get-standard-crypto-format {:market-values-per-currency market-values-per-currency}
                                                token-units)
              "0.005")))
+    (let [market-values-per-currency {:usd {:price nil}}
+          token-units                (money/bignumber 0.0123456)]
+      (is (= (utils/get-standard-crypto-format {:market-values-per-currency market-values-per-currency}
+                                               token-units)
+             "0.012346")))
     (let [market-values-per-currency {:usd {:price 0.005}}
           token-units                (money/bignumber 0.01)]
       (is (= (utils/get-standard-crypto-format {:market-values-per-currency market-values-per-currency}

--- a/status-go-version.json
+++ b/status-go-version.json
@@ -3,7 +3,7 @@
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "v0.184.45",
-    "commit-sha1": "9cdfd6fb42da23b0d5cae6e15548c26179c971ec",
-    "src-sha256": "0lsjzq20mg248l9ksa1fa59yq08npygrh0ak32khdi9s60bh1x94"
+    "version": "v0.184.46",
+    "commit-sha1": "9e7d13039e50eeb815e7fa7082080d2ec5c81493",
+    "src-sha256": "02f8s7bfq8wavdkqg4wpraan4j3vi1p8h1r22ngzp9nxfwjif8xr"
 }

